src/markets/get-disputable-markets.js

```diff
@@ -1,7 +1,22 @@
+/**
+ * @todo Update function description & return information once function has been implemented.
+ */
+
 "use strict";
 
 var augurNode = require("../augur-node");
 
+/**
+ * This function has not been implemented yet. Returns the markets in a specific reporting window that are able to be disputed, along with the value of the dispute bond needed to dispute each market’s result. Requires an Augur Node connection.
```

Comment:
Given "Dispute-Only Reporting System" this is going to change pretty significantly

Reply:
Yeah, that's fine. This can be left as-is for now and updated later.